src/fetchers/fetchBankHolidays.ts

```diff
@@ -0,0 +1,32 @@
+import moment from 'moment';
+import sdk from '@api/bamboohr'; // @bamboohr/v1#43haj4kl796vw3d
```

Comment:
Maybe remove this and similar comments in the other fetcher?

Reply:
Done!